Extensions/xcloud.js

```diff
@@ -813,23 +902,23 @@ XKit.extensions.xcloud = new Object({
 
 	md5: function(str) {
 
-		// http://kevin.vanzonneveld.net
-		// +   original by: Webtoolkit.info (http://www.webtoolkit.info/)
-		// + namespaced by: Michael White (http://getsprink.com)
-		// +    tweaked by: Jack
-		// +   improved by: Kevin van Zonneveld (http://kevin.vanzonneveld.net)
-		// +      input by: Brett Zamir (http://brett-zamir.me)
-		// +   bugfixed by: Kevin van Zonneveld (http://kevin.vanzonneveld.net)
-		// -    depends on: utf8_encode
-		// *     example 1: md5('Kevin van Zonneveld');
-		// *     returns 1: '6e658d4bfcb59cc13f96c14450ac40b9'
-		var xl;
-
-		var rotateLeft = function (lValue, iShiftBits) {
+		  // http://kevin.vanzonneveld.net
```

Comment:
The section starting here has a large amount of introduced whitespace errors (two spaces instead of \t)